contrib/automation_tests/orbit_top_down.py

```diff
@@ -75,12 +75,7 @@ def main(argv):
 
     logging.info('Re-listing items of the top-down view...')
     tree_items = main_wnd.TreeView.children(control_type='TreeItem')
-    row_count_after_expansion = len(tree_items) / TOP_DOWN_ROW_CELL_COUNT
 
-    if row_count_after_expansion != row_count_before_expansion + 2:
-        raise RuntimeError(
-            'First item of the top-down view doesn\'t have exactly two children'
-        )
```

Comment:
What do you think about adjusting this instead of removing it to keep the "at least two children" assertion explicit? E.g.:

```
     row_count_after_expansion = len(tree_items) / TOP_DOWN_ROW_CELL_COUNT	


    if row_count_after_expansion < row_count_before_expansion + 2:	
        raise RuntimeError(	
            'First item of the top-down view doesn\'t have at least two children'	
        )
```

Reply:
Changed as suggested